android/assets/jsons/translations/German.properties

```diff
@@ -210,7 +210,7 @@ Number of City-States = Anzahl Stadtstaaten
 One City Challenge = Herausforderung eine Stadt
 No Barbarians = Keine Barbaren
 No Ancient Ruins = Keine alten Ruinen
-No Natural Wonders = Keine natürlichen Wunder
+No Natural Wonders = Keine Natürlichen Wunder
```

Comment:
Nur so zur Anregung: Warum? Kein Grund in der Sprache (oder?) und dann ein Unterschied zu den "alten Ruinen"... Englisch muß nicht Vorbild sein?

Reply:
Ich sehe es als Eigenname und daher habe ich es entsprechend den anderen Übersetzungen angepasst. Dies war die einzige Stelle, an der "natürlichen" noch klein geschrieben war. Grundsätzlich hast du recht und die englische Variante muss nicht immer Vorbild sein.
Würde dies für dich als Eigenname zählen?

Reply:
Tja das ist die Frage... Suchmaschine hat für klein-n eine Mehrheit, wenn man Treffer ignoriert, wo das ganze ein Buchtitel oder Satzanfang ist. Eigentlich egal in diesem Umfeld - geben wir Konsistenz den Vorrang.